docs/competing_risks.md

```diff
@@ -0,0 +1,278 @@
+
+.. code-block:: python
+
+	# Obtain the survival probability
+	survival_modeler = LGBSurvivalModeler(data=mdata.drop('outcome', axis = 1))
+	survival_modeler.build_model(parallelize=False)
+	survival_modeler_forecasts = survival_modeler.forecast()
+	
+	# Set custom column names
+	custom_forecast_headers = date_range(data["year-month"].max(),
+	                                      periods=len(survival_modeler_forecasts .columns),
+	                                      freq="M").strftime("%b %Y")
+	survival_modeler_forecasts.columns = custom_forecast_headers
+	survival_modeler_forecasts
+	```
+	                           Jul 2020  Aug 2020  Sep 2020  Oct 2020  ...  May 2040  Jun 2040  Jul 2040  Aug 2040
+	country-leader                                                     ...                                        
+	Afghanistan: Ashraf Ghani  0.986044  0.965914  0.953822  0.942614  ...  0.000322  0.000320  0.000273  0.000271
+	Albania: Rama              0.992290  0.984483  0.977156  0.968703  ...  0.008364  0.008308  0.008252  0.008197
+	Algeria: Tebboune          0.991351  0.977605  0.965622  0.941865  ...  0.007509  0.007316  0.006988  0.006939
+	Andorra: Espot Zamora      0.990101  0.981595  0.975345  0.966500  ...  0.000727  0.000723  0.000718  0.000713
+	Angola: Lourenco           0.985230  0.972513  0.959758  0.947597  ...  0.025139  0.024493  0.023396  0.023232
+	                            ...       ...       ...       ...  ...       ...       ...       ...       ...
+	Venezuela: Nicolas Maduro  0.993150  0.986412  0.982183  0.975254  ...  0.000331  0.000329  0.000321  0.000313
+	Vietnam: Phu Trong         0.991983  0.982679  0.968412  0.959021  ...  0.002620  0.002586  0.002209  0.002123
+	Yemen: Houthi              0.992832  0.983635  0.950930  0.939746  ...  0.056119  0.055744  0.055367  0.054997
+	Zambia: Lungu              0.993150  0.986317  0.981646  0.973240  ...  0.008273  0.006682  0.005709  0.005411
+	Zimbabwe: Mnangagwa        0.993150  0.986421  0.981954  0.974727  ...  0.058133  0.057746  0.057357  0.056975
+	
+	[194 rows x 242 columns]
+	```
+
+##### Type of exit probabilities
+
+The same data can be used to train and forecast with the exit modeler, `LGBExitModeler`, where the terminal event type column name is passed as an argument to `exit_col`. The exit modeler produces forecasts for type of exit, conditional on :math:`t` being the last period observed. This is defined formally as
+
+.. math::
+   Pr(D_i=d_i|T_i= t,X_{i}).
+
+`LGBExitModeler` follows similar rules as `LGBSurvivalModeler` for selecting observations for training and forecasting. Except for training, where only observations with an observed exit in the desired time period are used.
+
+.. code-block:: python
```

Comment:
This code block also lacks syntax highlighting.

Reply:
Corrected. I used incorrect notation for comment blocks.